packages/flutter_tools/pubspec.yaml

```diff
@@ -14,15 +14,15 @@ dependencies:
   args: 2.5.0
   browser_launcher: 1.1.1
   dds: 4.2.5
-  dwds: 24.0.0
+  dwds: 24.1.0
   completion: 1.0.1
   coverage: 1.9.0
   crypto: 3.0.3
   ffi: 2.1.2
   file: 7.0.0
   flutter_template_images: 4.2.0
   html: 0.15.4
-  http: 0.13.6
+  http: 1.2.2
```

Comment:
FYI @andrewkolos

Reply:
https://pub.dev/packages/http/versions/0.13.6 - 15 months old!

Reply:
yep, Andrew and I had chatted about this and were concerned. Thanks for the fix!